common/setup.py

```diff
@@ -1,13 +1,25 @@
 import os
 from setuptools import find_packages, setup
+import re
 
 # allow setup.py to be run from any path
 os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
 
+# code found at: https://stackoverflow.com/a/7071358/6095378
+VERSIONFILE = "data_refinery_common/_version.py"
+verstrline = open(VERSIONFILE, "rt").read()
+VSRE = r"^__version__ = ['\"]([^'\"]*)['\"]"
```

Comment:
Not sure whether you should add more comments. I always found code with regular expression hard to follow.